client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java

```diff
@@ -363,15 +375,91 @@ protected PageReference getPageReference() {
         return panel;
     }
 
-    public class PlainSchemas extends Schemas {
+    public class PlainSchemasMemberships extends PlainSchemas<AttributableTO> {
```

Comment:
Any point to have this class and its sister class as public? Why not `protected static`, or at least `protected` if needed?

Reply:
Yes `protected` seems better!